docs/infrastructure.md

```diff
@@ -0,0 +1,87 @@
+## Infrastructure
+
+### [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process)
+
+### Infrastructure Overview
+The Google Infrastructure is managed via Terraform and nearly all the kubenetes deployments and such are done via a Helm Chart. See [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process) for more in depth guide. Datarepo is deployed on Kubernetes, Google Cloud Sql (postgres 11) and contains the following containers:
+
+- api (datarepo api container)
+- ui (datarepo ui container)
+- open id connect proxy (externally exposed endpoint proxy)
+- google cloud sql proxy (sql proxy to database)
+
+Container monitoring is done by [promethues operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator) this includes:
+
+- alertmanager (alertmanager can group send alerts)
+- grafana (graphical ui charts/graphs)
+- state-metric (collects state metrics)
+- node-exporter (collects k8 node metrics)
+- promethues (backend for metrics)
+
+Continuous deployment is managed by [Argocd](https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd) this includes:
+- dex (sso authentication container)
+- argo-repo-server (repo server)
+- argo-server (ui server)
+- redis (in-memory data structure store)
+- argo-application-controller
+
+### Helm Overview
+#### Repos
+- [Datarepo-helm](https://github.com/broadinstitute/datarepo-helm)
+  - Contains datarepo specific charts api, ui, proxy
+- [monster-helm](https://github.com/broadinstitute/monster-helm)
+  - Contains gcp managed cert chart
+- [argo-helm](https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd)
+  - Contains argocd chart for continuous deployment
+- [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
+  - Contains helm stable which includes the prometheus-operator chart
+
+#### Definitions
+This is specifically referring to a helm `Values.yaml` file. This file contains all the user specific values to be overritten in the chart being used. Datarepo uses an umbrella chart and each users values can be found [here](https://github.com/broadinstitute/datarepo-helm-definitions)
```

Comment:
So my `ddDeployments.yaml` is a values file? I didn't know that.

Reply:
yep it overrides the default values.yaml within the chart, if you ever wanna see what can be overwritten just look at the helm charts default value.yaml